packages/contracts/deployments/engine/deployment-config.template.ts

```diff
@@ -11,36 +11,15 @@ export const deployDetailsArray = [
     // the following can be undefined if you are using an existing admin ACL, otherwise define the Admin ACL contract name
     // if deploying a new AdminACL
     adminACLContractName: "AdminACLV1",
-    // See the `KNOWN_ENGINE_REGISTRIES` object in `/scripts/engine/V3/constants.ts` for the correct registry address for
-    // the intended network and the corresponding deployer wallet addresses
-    // @dev if you neeed a new engine registry, use the `/scripts/engine/V3/engine-registry-deployer.ts` script
-    engineRegistryAddress: "0x263113c07CB69eE047E6572E135E8C3C6302feFE",
-    randomizerContractName: "BasicRandomizerV2",
     genArt721CoreContractName: "GenArt721CoreV3_Engine",
     tokenName: "Engine Partner",
     tokenTicker: "PRTNR",
     startingProjectId: 0,
     autoApproveArtistSplitProposals: true,
     renderProviderAddress: "deployer", // use either "0x..." or special "deployer" which sets the render provider to the deployer
     platformProviderAddress: "deployer", // use either "0x..." or special "deployer" which sets the render provider to the deployer
-    // minter suite
-    minterFilterContractName: "MinterFilterV1",
-    minters: [
-      // include any of the most recent minter contracts the engine partner wishes to use
-      // @dev ensure the minter contracts here are the latest versions
-      "MinterSetPriceV4",
-      "MinterSetPriceERC20V4",
-      "MinterDAExpV4",
-      "MinterDAExpSettlementV1",
-      "MinterDALinV4",
-      "MinterHolderV4",
-      "MinterMerkleV5",
-    ],
     // set to true if you want to add an initial project to the core contract
-    addInitialProject: true,
-    // set to true if you want to add an initial token to the initial project
-    // (this will only work if you have set addInitialProject to true, and requires a MinterSetPriceV[4-9])
-    addInitialToken: true,
+    addInitialProject: false,
```

Comment:
removed addInitialToken because the security of trying to automatically figure out a minter to use on the shared minter filter vs. a minter being deployed alongside the core contract seemed too risky. It also seems to be rarely used functionality.

Reply:
> removed addInitialToken because the security of trying to automatically figure out a minter to use on the shared minter filter vs. a minter being deployed alongside the core contract seemed too risky. It also seems to be rarely used functionality.

This makes sense to me 👍 🙏